.mise/tasks/infrastructure/down

```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env sh
 
+#!/usr/bin/env sh
```

Comment:
I'm still confused why I'm seeing two of this line in GitHub...?

Reply:
fixed